### ISSUE TITLE
reproduce your code.

### ISSUE THREAD

**Issue by chenrxi** [jinglin-liang/DDDR#1]
Sorry, I try to reproduce the code on CUB200/Flowers102, but I failed. Can you give some help.

**Comment by jinglin-liang** [comment]
Could you provide more information? For example, is it a runtime error in the code, or are the final metrics not meeting expectations?

**Comment by chenrxi** [comment]
thank you for your reply and sorry for the missing details. I reproduce the code on CUB200 as I do recently and generate the image with noise, and it is very unsimilar to the real one. the batch size is 4 and dimension is 256, 10 rounds as you did in your code.  May be generating on CUB200 is much more difficult than cifar or tinyimagenet and the local **loss** is not converaged which is about 0.1to0.3. I want to ask the loss is normal or not?

**Comment by jinglin-liang** [comment]
I'm sorry, I haven't conducted experiments on this dataset, so I'm not sure if this loss is normal. However, after reviewing samples from CUB200, I believe our method could achieve good generation results. I think the issue is most likely due to hyperparameters. Since you've set the batch size to 4, which is smaller than our default of 12, I suggest increasing the number of local training steps or communication rounds.
